### ISSUE TITLE
Imported items from the library should land to the board as a group

### ISSUE THREAD

**Issue by lipis** [excalidraw/excalidraw#2186]
When using the library from mobile (If you try the link from #2179) it's quite hard to move the objects around if you miss-click (tap) and the inserted item.

**Comment by dwelle** [comment]
That would mean we'd be modifying what you actually saved into the library. You might want it to be grouped, and you might now. I'd leave that to the user to decide.

**Comment by lipis** [comment]
If there are groups within the item it's not a problem.. and it's easy to ungroup, double click or do stuff with it.. it is still intuitive, and it feels more like an imported item from the library as a whole.

**Comment by petehunt** [comment]
fwiw, i intentionally didn't do this but i can see the appeal and i'm not sure i made the right call :)

i think the groups feature is still not super discoverable. users might not know that they can edit parts of the objects from the library which would make the feature a lot less useful for them. just my 2 cents though.

**Comment by dwelle** [comment]
> users might not know that they can edit parts of the objects from the library which would make the feature a lot less useful for them.

Strong argument, especially since many users aren't even aware of grouping.

**Comment by lipis** [comment]
I disagree as the group discoverability, as I believe that group function exist in every app that creates diagrams (Figma, Adobe Illustrator, Inkscape, etc)

Double click is the most intuitive, but we also have buttons and the group thingy in the right click menu.
